source/monitoring.txt

```diff
@@ -200,9 +359,23 @@ Sink Connector JMX Metrics
      - The total number of failed batch writes attempted by the MongoDB sink
        task with a duration that exceeded 10000 milliseconds.
 
+.. _kafka-monitoring-source-attributes:
+
 Source Connector JMX Metrics
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. note:: Initial Commands and ``getMore`` Commands
+
+   Some metrics for source connector tasks distinguish between
+   **initial commands** and ``getMore`` commands. An initial
+   command is a ``find`` or ``aggregate`` command sent to a MongoDB
+   instance that retrieves the first set of documents in a client-side
+   MongoDB cursor. The ``getMore`` command is the MongoDB command that fetches
+   the subsequent sets of documents in a cursor.
+
+   To learn more about ``getMore`` commands, see the
+   :ref:`<manual-reference-commands-getMore>` page.
```

Comment:
I am aware this link is currently broken. Am looking into the cause of this, and should not effect technical review (just know this is supposed to go [here](https://www.mongodb.com/docs/manual/reference/command/getMore/)).